deploy/streisand-existing-cloud-server

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+#
+# Provision an existing cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# streisand-existing-cloud-server \
+#   --provider digitalocean \
```

Comment:
I don't think the `existing-cloud-server` should take a `--provider` arg for digitalocean. Copy pasta?